userguide/system.rst

```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
 
- sedutil-cli --scan
+Deploying SEDs
+^^^^^^^^^^^^^^
 
-In the results:
+To determine which devices support SED and their device names, go to the
+:menuselection:`Shell`
+and enter :command:`sedutil-cli --scan`. In the results:
 
 * **no** indicates a non-SED device
 * **1** indicates a legacy TCG OPAL 1 device
 * **2** indicates a modern TCG OPAL 2 device
 * **E** indicates a TCG Enterprise device
 
-To specify a password for a device, go to
-:menuselection:`Storage --> Disks`. Click |ui-options| on the
-confirmed SED device, then :guilabel:`Edit`. Enter and confirm the password
-in the :guilabel:`SED Password` and
-:guilabel:`Confirm SED Password` fields. Disks that have a configured
-password show bullets in their row of the
-:guilabel:`SED Password` column of :menuselection:`Storage --> Disks`.
-Conversely, the rows in that column will be empty for disks that do
-not support SED or which are unlocked using the global password.
+Example:
+
+.. code-block:: none
+
+   root@truenas1:~ # sedutil-cli --scan
+   Scanning for Opal compliant disks
+   /dev/ada0  No  32GB SATA Flash Drive SFDK003L
+   /dev/ada1  No  32GB SATA Flash Drive SFDK003L
+   /dev/da0   No  HGST    HUS726020AL4210  A7J0
+   /dev/da1   No  HGST    HUS726020AL4210  A7J0
+   /dev/da10    E WDC     WUSTR1519ASS201  B925
+   /dev/da11    E WDC     WUSTR1519ASS201  B925
+
+
+%brand% supports setting a global password for all detected SEDs or
+setting individual passwords for each SED. Using a global password for
+all SEDs is strongly recommended to simplify deployment and avoid
+physically maintaining separate passwords for each SED.
+
+
+**Setting a global password for SEDs**
+
+Go to
+:menuselection:`System --> Advanced --> SED Password`
+and enter the password. **Record this password and store it in a safe
+place!**
+
+Now the SEDs must be configured with this password. Go to the
+:menuselection:`Shell`
+and enter :samp:`sedhelper setup {password}`, where *password* is the
+global password entered in
+:menuselection:`System --> Advanced --> SED Password`.
 
-Remember to take ownership of the devices:
+:command:`sedhelper` ensures that all detected SEDs are properly
+configured to use the provided password. Example:
 
 .. code-block:: none
 
- sedhelper setup password
+   root@truenas1:~ # sedhelper setup abcd1234
+   da9			[OK]
+   da10			[OK]
+   da11			[OK]
+
+
+Rerun :samp:`sedhelper setup {password}` every time a new SED is placed
+in the system to apply the global password to the new SED.
+
+
+**Creating separate passwords for each SED**
+
+Go to
+:menuselection:`Storage --> Disks`.
+Click |ui-options| for the confirmed SED, then :guilabel:`Edit`.
+Enter and confirm the password in the :guilabel:`SED Password` and
+:guilabel:`Confirm SED Password` fields. Disks that have a configured
+SED password show bullets in their row of the :guilabel:`SED Password`
+column of
+:menuselection:`Storage --> Disks`.
+Conversely, the rows in that column will be empty for disks that do not
```

Comment:
Please don't do this.  It's the same as "alternatively", a negative that forces the reader to stop and think oppositely.  It is not needed here:
"Disks that do not support SED are not marked in this column."

"Support SED" is still an odd way to word it ("support self-encrypting devices" is misleading).  It can probably be said differently, like "support Opal encryption" or something.  We need to research this.